source/_integrations/zha.markdown

```diff
@@ -241,46 +241,32 @@ custom_quirks_path:
   type: string
 {% endconfiguration %}
 
-### OTA firmware updates
+### OTA updates of Zigbee device firmware
 
-The ZHA {% term integration %} has the ability to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware provider source URL for updates is available. OTA firmware updating is set to disabled (`false`) in the configuration by default.
+The ZHA integration have the ability to perform firmware updates of some specific Zigbee devices if they both support the standard Zigbee OTAU (Over-The-Air Update) feature and firmware images with standard Zigbee OTA format that brand and device are provided to the public by the manufacturer. You will get notifications if and when a device has firmware update available and to start an upgrade you have to initiate it by click a button.
```

Comment:
Maybe worth to mention that updates can be skipped as well.

Reply:
Please re-review latest commit with rewrite https://github.com/home-assistant/home-assistant.io/pull/28895/commits/50f9fe399bb5f94701f56a6bb857f6e23fde3603